src/components/DashboardMigrate.js

```diff
@@ -0,0 +1,48 @@
+import { ArrowLongRightIcon } from '@heroicons/react/24/outline';
+
+const DashboardMigrate = ( {} ) => {
+	const connectMigrate = () => {
+		// eslint-disable-next-line no-console
+		getMigrateRedirectUrl().then( ( res ) => {
+			// eslint-disable-next-line no-console
+			window.open( res?.data?.redirect_url, '_self' );
+		} );
+	};
+	return (
+		<div className=" nfd-rounded nfd-border nfd-p-4 nfd-grid nfd-grid-cols-2 nfd-gap-4">
+			<div className="nfd-grid nfd-grid-rows-[repeat(2,_min-content)] nfd-gap-6">
+				<div>
+					<h1 className="nfd-title nfd-title--2">
+						{
+							"	Let's migrate your existing site to your new account"
```

Comment:
Whats with the tab a spacing here? We should wrap this in translation functions.